catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergTable.java

```diff
@@ -47,12 +48,32 @@ public class IcebergTable extends BaseTable {
   /** The default provider of the table. */
   public static final String DEFAULT_ICEBERG_PROVIDER = "iceberg";
 
+  /** The supported file formats for Iceberg tables. */
+  public static final String ICEBERG_PARQUET_FILE_FORMAT = "parquet";
+
```

Comment:
remove blank line?

Reply:
> remove blank line?

When spotlessApply is executed, empty lines are automatically added.
Maybe we can add comments to each format？

Reply:
both ok for me